tests/test_app.py

```diff
@@ -29,10 +29,9 @@
 from flask_oauthlib.client import OAuth as FlaskOAuth
 from flask_oauthlib.client import OAuthRemoteApp
 from invenio_db import InvenioDB, db
-from sqlalchemy_utils.functions import create_database, database_exists
-
 from invenio_oauthclient import InvenioOAuthClient
 from invenio_oauthclient.contrib.orcid import REMOTE_APP
+from sqlalchemy_utils.functions import create_database, database_exists
```

Comment:
First try to completely uninstall `sqlalchemy_utils`. You might need to run `pip uninstall sqlalchemy-utils` multiple times. Then you can try to install it from pypi and run the isort again.